insights/parsers/ansible.py

```diff
@@ -0,0 +1,61 @@
+"""
+Ansible - Parsers relates to Ansible
+====================================
+
+Below Parser is included in this module
+
+AnsibleTelemetry - command "/usr/share/ansible/telemetry/telemetry.py"
+----------------------------------------------------------------------
+"""
+
+from insights import CommandParser, JSONParser, parser
+from insights.specs import Specs
+
+
+@parser(Specs.ansible_telemetry)
+class AnsibleTelemetry(CommandParser, JSONParser):
```

Comment:
Does this inheritance provide the ability to handle newline deleimited JSON?

The output from the command will produce multiple JSON documents split by newlines.

Imagine instead of the pretty printed data below, it is a single line, followed by a similar JSON document on the next line.

Reply:
Thanks for point that out, I mis-understood the `NL-JSON` mentioned the Jira ticket. This super class cannot handle the NDJSON. I'll update a bit to make it work as expected.